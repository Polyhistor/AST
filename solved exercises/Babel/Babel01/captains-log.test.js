// babel exercise 0 (captains-log)
// When you're finished with this exercise, run
//   "npm start exercise.babel.1"
//   to move on to the next exercise

import pluginTester from 'babel-plugin-tester';
import captainsLog from './captains-log';

pluginTester({
  plugin: captainsLog,
  snapshot: true,
  tests: [
    { code: `anything.log();`, snapshot: false },
    `console.log('sup dawg')`
  ]
});
