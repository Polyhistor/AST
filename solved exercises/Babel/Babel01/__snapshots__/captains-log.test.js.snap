// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`2. console-text-replacer 1`] = `
"
console.log('sup dawg')

      ↓ ↓ ↓ ↓ ↓ ↓

console.log('1:0');
"
`;
